### PR TITLE
PD-362 NFS ports

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

**Comment by nathan-px** [comment]
adding @vilasd . Vilas, Aditya said you may be able to help out in validating this.

File: content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-sharedv4-pvcs.md [hunk 1]
```diff
@@ -8,7 +8,11 @@ series: k8s-vol
 
 This document describes how to use Portworx **sharedv4** (ReadWriteMany) volumes in your Kubernetes cluster.
 
-#### Provision a Sharedv4 Volume
+## Prerequisites
```
Comment (adityadani) [inline content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-sharedv4-pvcs.md:11]: Can we also link from our Install requirements ports section ? https://docs.portworx.com/start-here-installation/#installation-prerequisites

File: content/portworx-install-with-kubernetes/storage-operations/create-pvcs/open-nfs-ports.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+    rpcinfo -p
+    ```
+
+SharedV4 volumes communicate over the following ports:
+
+* portmapper: 111 (default on all linux distributions)
+* nfs service: 2049 (default on all linux distributions)
+* mountd: 20048 (depends on the linux distribution)
+
+If the default ports your OS uses match these ports, proceed to the **Open NFS ports on most Linux distributions** section.
```
Comment (adityadani) [inline content/portworx-install-with-kubernetes/storage-operations/create-pvcs/open-nfs-ports.md:32]: nit: these could be links to the subsequent sections.

File: content/portworx-install-with-kubernetes/storage-operations/create-pvcs/open-nfs-ports.md [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+```text
+iptables -I INPUT -p tcp -m tcp --match multiport --dports 111,2049,20048 -j ACCEPT
+iptables -I OUTPUT -p tcp -m tcp --match multiport --dports 111,2049,20048 -j ACCEPT
+```
+
+Once you've determined your hosts are using the default ports and opened those ports, you can start using sharedV4 volumes.
+
+## Manually assign and open NFS ports
+
+For certain Linux distributions, the OS chooses the `mountd` randomly every time the node reboots. To solve this, you must manually assign NFS ports, and how you accomplish this depends on your OS.
```
Comment (adityadani) [inline content/portworx-install-with-kubernetes/storage-operations/create-pvcs/open-nfs-ports.md:51]:
```suggestion
For certain Linux distributions, the OS chooses the `mountd` port randomly every time the node reboots. To solve this, you must manually assign NFS ports, and how you accomplish this depends on your OS.
```

**Comment by adityadani** [comment]
@nathan-px when are you planning to merge this PR ?

File: content/start-here-installation.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ The minimum supported size for a Portworx cluster is three nodes. Each node must
 
 |**Network** ||
 |--- | ---|
-|Open needed ports | TCP ports 9001-9022 and UDP port 9002 on all Portworx nodes. Also open the KVDB port. \(As an example, `etcd` typically runs on port 2379\)|
+|Open needed ports | TCP ports 9001-9022 and UDP port 9002 on all Portworx nodes. Also open the KVDB port. \(As an example, `etcd` typically runs on port 2379\). If you intend to use Portworx with sharedV4 volumes, you may need to [open your NFS ports](/portworx-install-with-kubernetes/storage-operations/create-pvcs/open-nfs-ports).|
```
Comment (nathan-px) [inline content/start-here-installation.md:25]: @adityadani How's this?

Comment (adityadani) [reply content/start-here-installation.md:25]: looks good

**Pull request merged**